package.json

```diff
@@ -23,6 +23,7 @@
     "publish-release": "yarn lerna publish --conventional-graduate",
     "publish-prerelease": "yarn lerna publish prerelease --preid beta --dist-tag next",
     "test": "jest",
+    "test-cli": "node ./packages/acceptance-tests/run-tests",
```

Comment:
Might want to use [lerna run](https://github.com/lerna/lerna/tree/main/commands/run#readme) here.